libs/ui-components/src/DocumentType/DocumentTypeGenericList.tsx

```diff
@@ -8,15 +8,16 @@ interface TableOverviewGenericProps {
   isLoading?: boolean;
 }
 
-export function DocumentTypeGenericList({documents, documentType, isLoading}: TableOverviewGenericProps) {
+export function DocumentTypeGenericList({documents, isLoading}: TableOverviewGenericProps) {
   return (
     <Table
       isLoading={isLoading}
       headers={["Title", "Created", "Status"]}
       rows={documents.map((document, key) => [
         <Link key={`${key}-${document.id}-id`} href={`/content/${document.documentType}/${document.id}`}>
-          <p className="font-medium text-black dark:text-white">{document.data[documentType.titleField] as string}</p>
+          <p className="font-medium text-black dark:text-white">{(document.data.title as string) || ""}</p>
```

Comment:
The data field that represents "title" should be generic and defined by its document type. For example, in an article it makes most sense ot display `title` and the title. But when listing authors, it might make more sense to use the field `name` as a title in a list.

Keep the original implementation.

Reply:
Done ✅